src/utils/src/Thread.c

```diff
@@ -37,7 +37,7 @@ PUBLIC_API DWORD WINAPI startWrapperRoutine(LPVOID data)
     return retVal;
 }
 
-PUBLIC_API STATUS defaultCreateThread(PTID pThreadId, startRoutine start, PVOID args)
+PUBLIC_API STATUS defaultCreateThreadWithParams(PTID pThreadId, startRoutine start, SIZE_T stackSize, PVOID args)
```

Comment:
shouldn't this one use latest param struct defn?

Reply:
The PR adjusting that has just been merged.